spec/services/tip_options_service_property_spec.rb

```
@@ -0,0 +1,168 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'prop_check'
+
+RSpec.describe TipOptionsService, type: :service do
+  include PropCheck::RSpec
+
+  describe 'are_tip_options_valid?', :property_based do
```

RuboCop (1.75.5)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)